src/pfe/portal/modules/User.js

```diff
@@ -53,6 +58,12 @@ module.exports = class User {
       this.uiSocket = uiSocket;
     }
     this.secure = true;
+    this.dockerConfigFile = "/root/.docker/config.json";
+    this.codewindPFESecretName = "codewind-pfe-docker-registries";
+    // Get the Kube Client context when running in K8s
+    if (global.codewind.RUNNING_IN_K8S == true) {
+      this.k8client = new K8Client({ config: k8config.getInCluster(), version: '1.9' });
```

Comment:
Not sure about this. Will need to check. Portal is already doing it this way in their server.js

Reply:
So I had initially had the Kube client initialized in a single place(ie Portal and Portal was telling Turbine during PFE startup about the Kube Client obj) across all the Codewind codebase but after thinking about it, i think it is cleaner and better if Portal & Turbine had their own Kube Client initialization since they're different modules.

So I have updated the Portal code and I am only using the initialized Kube Client and not doing any re-initialization. The change is in the commit https://github.com/eclipse/codewind/pull/1118/commits/0ebb283e946cb4f21fe18051068a806024e9be30